apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Yggdrasil.User do
   use Ecto.Schema
   import Ecto.Changeset
+  import Ecto.Query, only: [from: 1, from: 2] # dislike this
```

Comment:
I dislike that comment :facepunch:. So you don't like that you have to import each `from`?